client/js/components/App.react.jsx

```diff
@@ -1,71 +1,63 @@
-import $ from 'jquery';
 import React from 'react';
-import { GeoJson, Map, Marker, Popup, TileLayer } from 'react-leaflet';
+
+import api from '../utils/api';
 import ParkMap from './ParkMap.jsx';
 import ParkList from './ParkList.jsx';
 
 
-function getParksJson() {
-    return $.getJSON("/data/parks.json");
-}
-
-function getFeatureGeoJson(parkID, featureType) {
-    return $.getJSON(`/data/${featureType}/park_${parkID}.geojson`);
-}
-
 export default class App extends React.Component {
 
     constructor(props) {
         super(props);
-        this.state = {park: null,
-                      parkList: [],
-                      parkByID: {}};
-        getParksJson().then((data) => {
-            var parkByID = {}
-            data.forEach((current, index) => {
-                // NOTE: Coordinates are lng lat here
-                current.center.reverse();
-                parkByID[current.park_id] = current;
-            });
-            this.setState({parkList: data, parkByID: parkByID});
-        });
+        this.state = {
```

Comment:
i believe setting this.state is highly discouraged/ will break react. define `this.getInitialState()` or  call `this.setState()`